docker-compose.test.yml

```diff
@@ -4,7 +4,7 @@ services:
 
     testldap:
       container_name: testldap
-      image: quay.io/prodsecdev/openldap:2.5.7-debian-10-r57
+      image: docker.io/bitnami/openldap:2.5.13-debian-11-r19
```

Comment:
maybe we should target `2.5`? from looking at https://hub.docker.com/r/bitnami/openldap/tags it seems like that's the least specific version that should cover any future security updates

Reply:
True, we should be more general. `2.5` or `2.5-debian...` though? :thinking:

Reply:
`2.5` should be fine